src/main/scala/cz/alenkacz/db/postgresscala/PostgresAsyncDbValue.scala

```diff
@@ -11,19 +11,25 @@ import scala.collection.mutable.ArrayBuffer
 class PostgresAsyncDbValue(value: Any) extends DbValue {
   override def string: String = value.asInstanceOf[String]
 
-  override def int: Int = value.asInstanceOf[Int]
+  def throwOnNull(value: Any): Any =
+    if (value == null)
+      throw new IllegalArgumentException("Trying to get value, got null instead. Use methods returning option for NULLable columns")
+    else
+      value
 
-  override def bigInt: Long = value.asInstanceOf[Long]
+  override def int: Int = throwOnNull(value).asInstanceOf[Int]
 
-  override def double: Double = value.asInstanceOf[Double]
+  override def bigInt: Long = throwOnNull(value).asInstanceOf[Long]
```

Comment:
It seemed that you chose naming according to Scala types and I like it :) So `bigInt` didn't fit.